documentation/docs/mock-apps/services/index.md

```diff
@@ -37,68 +40,63 @@ graph TD
 
 ## Config
 
-| Property | Required | Description | Type |
-|----------|----------|-------------|------|
-| name | Yes | The name of the service (depends on service used)| [Service](#available-services) |
-| parameters | Yes | An array of parameter objects for the services (depends on service used)| [Service](#available-services) |
+| Property   | Required | Description                                                              | Type                           |
+| ---------- | -------- | ------------------------------------------------------------------------ | ------------------------------ |
+| name       | Yes      | The name of the service (depends on service used)                        | [Service](#available-services) |
+| parameters | Yes      | An array of parameter objects for the services (depends on service used) | [Service](#available-services) |
 
 ## Example
 
 ```json
 {
-    "services": [
+  "services": [
+    {
+      "name": "processDPP",
+      "parameters": [
         {
-            "name": "processObjectEvent",
-            "parameters": [
-                {
-                    "vckit": {
-                        "vckitAPIUrl": "http://localhost:3332/v2",
-                        "issuer": "did:web:agtrace-passports.s3.ap-southeast-2.amazonaws.com"
-                    },
-                    "dpp": {
-                        "context": [
-                            "https://dpp-json-ld.s3.ap-southeast-2.amazonaws.com/dppld.json"
-                        ],
-                        "renderTemplate": [],
-                        "type": [
-                            "VerifiableCredential",
-                            " DigitalProductPassport"
-                        ],
-                        "dlrLinkTitle": "Steel Passport",
-                        "dlrIdentificationKeyType": "gtin",
-                        "dlrVerificationPage": "http://localhost:3000/verify"
-                    },
-                    "dlr": {
-                        "dlrAPIUrl": "http://localhost:8080",
-                        "dlrAPIKey": "5555555555555"
-                    },
-                    "storage": {
-                        "url": "http://localhost:3001/upload",
-                        "params": {
-                            "resultPath": "/url"
-                        },
-                        "options": {
-                            "method": "POST",
-                            "headers": {
-                                "Content-Type": "application/json"
-                            }
-                        }
-                    },
-                    "identifierKeyPath": "/product/itemIdentifiers/0/identifierValue"
-                }
-            ]
-        },
+          "vckit": {
+            "vckitAPIUrl": "http://localhost:3332/v2",
+            "issuer": "did:web:agtrace-passports.s3.ap-southeast-2.amazonaws.com"
```

Comment:
Should we change this issuer to an example?

Reply:
Updated :)